README.md

```diff
@@ -37,7 +37,7 @@ The distribution contains most of the sources as jar files. The source code of F
 
 ### JDK 8+
 
-Flowable runs on a JDK higher than or equal to version 8. Go to https://www.oracle.com/technetwork/java/javase/downloads/index.html and click on button "Download JDK".  There are installation instructions on that page as well. To verify that your installation was successful, run "java -version" on the command line.  That should print the installed version of your JDK.
+Flowable runs on a JDK higher than or equal to version 8. Use the JDK packaged with your Linux distribution or go to https://adoptopenjdk.net and click on button "Last version". There are installation instructions on that page as well. To verify that your installation was successful, run "java -version" on the command line.  That should print the installed version of your JDK.
```

Comment:
The button on https://adoptopenjdk.net is titled "Latest release" (using an en_US locale).

Reply:
yes you're right, I'll fix that ...